server/pages/index.jade

```diff
@@ -54,6 +54,8 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 				link(rel='stylesheet', href=urls['style-debug.css'])
 			else
 				link(rel='stylesheet', href=urls['style.css'])
+		style(type='text/css' id='critical-path-css')
+			!= Array.from(css).join('')
```

Comment:
How do we decide what is critical path CSS? From what I can tell, this is including every component's styles (currently just VerticalNav)?

Minor: Spacing between parentheses.